extensions/eclipse-che-theia-plugin-ext/package.json

```diff
@@ -24,6 +24,7 @@
     "vscode-uri": "2.1.1",
     "js-yaml": "3.13.1",
     "drivelist": "9.0.2",
+    "@eclipse-che/theia-remote-api": "^0.0.1",
     "@eclipse-che/workspace-telemetry-client": "latest"
   },
```

Comment:
`"@eclipse-che/workspace-client": "latest"` is redundant here?

Reply:
hello, I'm not sure to follow your comment ?

Reply:
I mean can we get rid of `workspace-client` dependency in `plugin-ext`?
https://github.com/eclipse/che-theia/blob/6926f0b8c2c4964ff47b1231cbc98ba44e6366f1/extensions/eclipse-che-theia-plugin-ext/package.json#L17
As `theia-remote-api` is going to be used.

Reply:
got it, yes sure, will fix in this PR
There will be other PR anyways. Probably splitting remote-api in two (to be sure to cut down transitive dependencies)

Reply:
I've removed workspace-client